src/wallet/wallet.h

```diff
@@ -77,7 +80,13 @@ enum WalletFeature {
     FEATURE_WALLETCRYPT = 40000, // wallet encryption
     FEATURE_COMPRPUBKEY = 60000, // compressed public keys
 
-    FEATURE_LATEST = 61000
+    PRE_PIVX = 61000, // inherited version..
+
+    FEATURE_HD = 130000, // Hierarchical key derivation after BIP32 (HD Wallet)
```

Comment:
since we're doing this in one go and not incrementally like in upstream, do we even need to have both `FEATURE_HD` and `FEATURE_HD_SPLIT`?

Reply:
nope we don't, check that there is no check for `FEATURE_HD` or `FEATURE_HD_SPLIT`. It directly checks for `IsHDEnabled()` everywhere.

I only left this `FEATURE_HD` there as merely a reminder for future back ports checks. But.. it's true that can just leave a good comment there and would be less confusing for future readers.